README.md

```diff
@@ -15,17 +15,30 @@ Before getting started, make sure you have a development workspace where you hav
 #### Environment Variables
 Before you can run the app, you'll need to store some environment variables.
 
-1. Copy `.env.sample` to `.env`
+1. Copy `env.sample` to `.env`
```

Comment:
Pondering: should we have them rename this file to `.env` instead?

Reply:
@misscoded Good q!! It seems that currently we have a `.gitignore` for all `.env*` files, so if we did this we'd likely need to update the `.gitignore` for all related projects to ignore all `.env*` files except `.env.sample`. We'd also need to update `env.sample`s for all related projects as well to reflect the rename.

I do feel like using `.env.sample` might be better practice generally and it aligns more with other projects I've seen, but I am open to keeping it if there's some sort of additional context supporting this being named `env.sample` when this template was introduced!

Reply:
Update on this - we decided to rename `.env.sample`, update the README instructions for folks to rename this file to `.env` when getting started, and also added `.env.sample` to the `.gitignore`! 💯